neuralmonkey/readers/plain_text_reader.py

```diff
@@ -1,16 +1,13 @@
 from typing import List, Iterable
 import gzip
 
-from neuralmonkey.readers.utils import FILETYPER
-
 
 def get_plain_text_reader(encoding: str = "utf-8"):
     """Get reader for space-separated tokenized text."""
     def reader(files: List[str]) -> Iterable[List[str]]:
         for path in files:
-            mime_type = FILETYPER.from_file(path)
 
-            if mime_type == "application/gzip":
+            if path.endswith(".gz"):
```

Comment:
Chtelo by to dopsat i do dokumentace (nekde v tutorialu to bylo, myslim), ze se to resi podle pripony a ne podle typu.